<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>

 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>520</width><height>500</height></rect>
  </property>
  <property name="windowTitle">
   <string>User Portal</string>
  </property>

  <widget class="QWidget" name="centralwidget">



   <layout class="QVBoxLayout" name="mainLayout">

    <item>
     <widget class="QLabel" name="label">
      <property name="text"><string>User Portal</string></property>
      <property name="alignment"><set>Qt::AlignCenter</set></property>
     </widget>
    </item>

    <item>
     <spacer name="topSpacer">
      <property name="orientation"><enum>Qt::Vertical</enum></property>
      <property name="sizeHint" stdset="0">
       <size><width>20</width><height>20</height></size>
      </property>
     </spacer>
    </item>

    <item><widget class="QPushButton" name="btnCreateAccount"><property name="text"><string>Create Account</string></property><property name="minimumHeight"><number>45</number></property></widget></item>
    
    <!-- NEW: Pay Loan Button (Initially Hidden via Code if needed, or controlled by logic) -->
    <item>
     <widget class="QPushButton" name="btnPayLoan">
      <property name="text"><string>Pay Loan</string></property>
      <property name="minimumHeight"><number>45</number></property>
      <!-- We can set it to invisible by default in python or use logic -->
     </widget>
    </item>

    <item><widget class="QPushButton" name="btnWithdraw"><property name="text"><string>Withdraw</string></property><property name="minimumHeight"><number>45</number></property></widget></item>
    <item><widget class="QPushButton" name="btnDeposit"><property name="text"><string>Deposit</string></property><property name="minimumHeight"><number>45</number></property></widget></item>
    <item><widget class="QPushButton" name="btnTransfer"><property name="text"><string>Transfer</string></property><property name="minimumHeight"><number>45</number></property></widget></item>
    <item><widget class="QPushButton" name="btnHistory"><property name="text"><string>Transaction History</string></property><property name="minimumHeight"><number>45</number></property></widget></item>

    <!-- NEW SHOW BALANCE BUTTON -->
    <item>
     <widget class="QPushButton" name="btnShowBalance">
      <property name="text"><string>Show Account Balance</string></property>
      <property name="minimumHeight"><number>45</number></property>
     </widget>
    </item>

    <item>
     <widget class="QPushButton" name="btnApplyLoan">
      <property name="text"><string>Apply for Loan</string></property>
      <property name="minimumHeight"><number>45</number></property>
     </widget>
    </item>

    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text"><string>Logout</string></property>
      <property name="minimumHeight"><number>45</number></property>
     </widget>
    </item>

    <item>
     <spacer name="bottomSpacer">
      <property name="orientation"><enum>Qt::Vertical</enum></property>
      <property name="sizeHint" stdset="0">
       <size><width>20</width><height>40</height></size>
      </property>
     </spacer>
    </item>

   </layout>
  </widget>
 </widget>

 <resources/>
 <connections/>
</ui>
